Doc/library/stdtypes.rst

```diff
@@ -4248,7 +4248,7 @@ pairs within braces, for example: ``{'jack': 4098, 'sjoerd': 4127}`` or ``{4098:
    value)`` pairs. Order comparisons ('<', '<=', '>=', '>') raise
    :exc:`TypeError`.
 
-   Dict preserves insertion order.  Note that updating key doesn't affects the
+   Dict preserves insertion order.  Note that updating a key does not affect the
    order.  On the other hand, keys added after deletion are inserted to the
    last. ::
```

Comment:
While you're in there: "inserted to the last" -> "inserted at the end" ?  Also, I notice that this most recent change uses "dict" whereas the rest of the section uses "dictionary".